source/aks/patching-dns-node-cache.html.md.erb

```diff
@@ -0,0 +1,19 @@
+---
+title: DNS Node Cache Patching
+weight: 30
+last_reviewed_on: 2024-02-26
+review_in: 12 months
+---
+
+# <%= current_page.data.title %>
+
+This document cover how to check and patch the DNS Node Cache image.
+The image should be updated automatically by renovate, but if it is not, you can manually update the image in the `nodelocaldns.yaml` file in the `cnp-flux-config` repo.
```

Comment:
Hi Dan, why not make this a link to the actual file

Reply:
Good idea, I have updated the PR.